src/test/skript/tests/syntaxes/expressions/ExprEntities.sk

```diff
@@ -0,0 +1,7 @@
+test "entities in chunk":
+	spawn 10 sheep at spawn of world "world"
+	wait 1 tick
+	assert size of all entities in chunk at spawn of world "world" > 10 with "Size of all entities in spawn chunk is not > 10: %size of all entities in chunk at spawn of world ""world""%"
```

Comment:
```suggestion
	assert size of all entities in chunk at spawn of world "world" >= 10 with "Size of all entities in spawn chunk is not >= 10: %size of all entities in chunk at spawn of world ""world""%"
```
If there are no entities before, the test will fail.

Reply:
Oh crap you are totally right, thanks for that.